client/businessconfig/src/main/java/org/opfab/businessconfig/model/AcknowledgmentAllowedEnum.java

```diff
@@ -32,4 +34,20 @@ public enum AcknowledgmentAllowedEnum {
     ONLY_WHEN_RESPONSE_DISABLED_FOR_USER("OnlyWhenResponseDisabledForUser");
 
     String value;
+
+    @Override
+    @JsonValue
+    public String toString() {
+        return String.valueOf(value);
+    }
+
+    @JsonCreator
+    public static AcknowledgmentAllowedEnum fromValue(String text) {
+        for (AcknowledgmentAllowedEnum b : AcknowledgmentAllowedEnum.values()) {
+            if (String.valueOf(b.value).equals(text)) {
+                return b;
+            }
+        }
+        return null;
```

Comment:
Why this is needed ?

Reply:
I can't figure out why I needed it. Weird...
(So I remove)